specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/sql.core_legacy.json

```diff
@@ -108,111 +108,6 @@
-          },
-          {
-            "$ref": "#/parameters/ServerNameParameter"
-          },
-          {
-            "name": "databaseName",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "description": "The name of the database for which setting the transparent data encryption applies."
-          },
-          {
-            "$ref": "#/parameters/TransparentDataEncryptionNameParameter"
-          },
-          {
-            "name": "parameters",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/TransparentDataEncryption"
-            },
-            "description": "The required parameters for creating or updating transparent data encryption."
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/TransparentDataEncryption"
-            }
-          },
-          "201": {
-            "description": "Created",
-            "schema": {
-              "$ref": "#/definitions/TransparentDataEncryption"
-            }
-          }
-        }
-      },
-      "get": {
-        "tags": [
-          "TransparentDataEncryption"
-        ],
-        "operationId": "TransparentDataEncryptions_Get",
-        "description": "Gets a database's transparent data encryption configuration.",
-        "x-ms-examples": {
-          "Get a database's transparent data encryption configuration": {
-            "$ref": "./examples/DatabaseTransparentDataEncryptionGet.json"
-          }
-        },
-        "parameters": [
-          {
-            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
-          },
-          {
-            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
-          },
-          {
-            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
-          },
-          {
-            "$ref": "#/parameters/ServerNameParameter"
-          },
-          {
-            "name": "databaseName",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "description": "The name of the database for which the transparent data encryption applies."
-          },
-          {
-            "$ref": "#/parameters/TransparentDataEncryptionNameParameter"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/TransparentDataEncryption"
-            }
-          }
-        }
-      }
-    },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/transparentDataEncryption/{transparentDataEncryptionName}/operationResults": {
       "get": {
```

Comment:
do we have impl to replace this TDE API? why we still left it?

Reply:
I have removed all TDE related config in sql.core_legacy.json